schedule.md

```diff
@@ -10,6 +10,7 @@ You can download the calendar to your device by clicking [here](https://calendar
 <iframe src="https://calendar.google.com/calendar/embed?height=1000&amp;wkst=2&amp;bgcolor=%234a578e&amp;ctz=Europe%2FMadrid&amp;src=YmhnLWRvbm9zdGlhQGJjYmwuZXU&amp;color=%23039BE5&amp;title=Brainhack%20Donostia%202020&amp;mode=AGENDA&amp;showTabs=0&amp;showCalendars=0&amp;showPrint=0&amp;tab=mc&mode=week&dates=20201109/20201115" style="border:solid 1px #777" width="100%" height="1000" frameborder="0" scrolling="no"></iframe>
 </div>
 
+*Times displayed on the calendar are automatically updated based on the user's time zone.
```

Comment:
Could you please write this above the iframe and without the *.

People will first see the text, so it's better that they see it's showing their own timezone before looking at the agenda.